README.md

```diff
@@ -1,82 +1,106 @@
 # README
-## Introduction
-This README contains description of the examples how to build and configure a validator node in `net.ton.dev` blockchain. The instructions and scripts below were verified on Ubuntu 18.04.
-## Getting Started
-### Prerequisites
-#### System Requirements
+
+This HOWTO contains instructions on how to build and configure a validator node in TON blockchain. The instructions and scripts below were verified on Ubuntu 18.04.
+# Getting Started
+
+## System Requirements
 | Configuration | CPU (cores) | RAM (GiB) | Storage (GiB) | Network (Gbit/s)|
 |---|:---|:---|:---|:---|
 | Minimal |2|8|1000|1|
 | Recommended |16|32|1000|1| 
-## Instructions
+## Prerequisites
 ### 1. Build Node
 Adjust (if needed) `net.ton.dev/scripts/env.sh`
 
 Build a node:
 
-    $ cd net.ton.dev/scripts/ && ./build.sh
+    $ cd ./scripts/ && ./build.sh
 ### 2. Setup Node
 Initialize a node:
 
-    $ cd net.ton.dev/scripts/ && ./setup.sh
-### 3. Run Node
+    $ cd ./scripts/ && ./setup.sh
+### 3. Prepare Multisignature Wallet
+Multisignature wallet (or just wallet) is used in validator script to send election requests to the Elector smart contract.
+
+Let `N` be the total number of wallet custodians and `K` the number of minimal confirmations required to execute a wallet transaction.
+
+1. Read [TONOS-CLI documentation](https://docs.ton.dev/86757ecb2/v/0/p/94921e-running-tonos-cli-with-tails-os-and-working-with-multisignature-wallet) (*Deploying Multisignature Wallet to TON blockchain*) and generate seed phrases and public keys for `N - 1`  custodians.
+2. Generate wallet address and `Nth` custodian key:
+```
+    $ cd ./scripts/ 
+    $ ./msig_genaddr.sh
+```
+Script creates 2 files: `$(hostname -s).addr` and `msig.keys.json` in `~/ton-keys/` folder. 
+Put them next to `validator_msig.sh` script. 
```

Comment:
Remove this - need to keep the keys where they are generated.

Reply:
done